bundles/pom.xml

```diff
@@ -520,5 +523,4 @@
       </build>
     </profile>
   </profiles>
-
```

Comment:
Can you leave this as-is?

Reply:
Sorry for this but this must somewhere some configuration problem in my git explorer because I don't see this in my offline version only in the online version

Reply:
Here, too.